src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Morphisms/Constructors.jl

```diff
@@ -1,3 +1,36 @@
+@doc raw"""
+    morphism_of_projective_schemes(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map; check::Bool=true )
+
+Given a morphism ``f : T → S`` of the `homogeneous_coordinate_ring`s of `Q` and `P`, respectively, 
+construct the associated morphism of projective schemes.
+"""
+function morphism_of_projective_schemes(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map; check::Bool=true )
+  return ProjectiveSchemeMor(P, Q, f, check=check)
+end
+
+@doc raw"""
+    morphism_of_projective_schemes(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map, h::SchemeMor; check::Bool=true )
+
+Suppose ``P ⊂ ℙʳ_A`` and ``Q ⊂ ℙˢ_B`` are projective schemes, ``h : Spec(A) → Spec(B)`` is a 
+morphism of their `base_scheme`s, and ``f : T → S`` a morphism of the 
+`homogeneous_coordinate_ring`s of `Q` and `P` over ``h* : B → A``.
```

Comment:
```suggestion
`homogeneous_coordinate_ring`s of `Q` and `P` over ``h^* : B → A``.
```
Otherwise this will render as a multiplication `*` in the docs.

Reply:
> Thank you @HechtiDerLachs - this looks good to me!
> 
> Nitpick: Personally, I benefit from examples and suffer from their absence. So, as is, I must suffer. Any chance to add examples?
> 
> (Of course, this extends the runtimes of the tests. But still, there should exist quick and simple examples, not?)

I think that examples are really helpful for constructors. Then I can copy paste them.
For properties and most attributes like e.g. `is_smooth` or `base_ring` I find them useless and the documentation bloated.

Reply:
So if I have a wish, please add some doctests in the constructors.